tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3161,7 +3142,10 @@ public function createTheGroup(string $group, ?string $method = null):void {
 				}
 				break;
 			case "graph":
-				$newGroup = $this->graphContext->adminHasCreatedGroupUsingTheGraphApi($group);
+				$newGroup = $this->graphContext->createGroup($group);
+				if ($newGroup->getStatusCode() === 201) {
```

Comment:
AFAIK, stable-4 will return `200`
```suggestion
				if ($newGroup->getStatusCode() === 200) {
```